dataland-frontend/tests/e2e/specs/framework-view-and-upload-journey/UploadFrameworkDataJourney.ts

```diff
@@ -219,16 +219,13 @@ describe('As a user, I expect the dataset upload process to behave as I expect',
       function checkIfDropDownSwitchRendersData(): void {
         cy.get('div[data-test="chooseFrameworkDropdown"]').click();
         cy.get("li:contains('EU Taxonomy for financial companies')").click();
-        cy.get('td[data-cell-label="Fiscal Year End"]').should('be.visible');
+        cy.get('td[data-cell-label="Level of Assurance"]').should('be.visible');
 
         cy.get('div[data-test="chooseFrameworkDropdown"]').click();
         cy.get("li:contains('LkSG')").click();
-        cy.get('td[data-cell-label="Data Date"]')
-          .should('be.visible')
-          .next('td')
-          .find('span')
-          .should('be.visible')
-          .contains('2022-07-30');
+        cy.get('td[data-cell-label="Data Date"]').should('be.visible');
+
+        cy.get('td[data-cell-label="Data Date"]').next('td').find('span').should('be.visible').contains('2022-07-30');
```

Comment:
Warum gibt es Änderungen in diesem Test?
Auf den ersten Blick scheint der Test nichts mit den Feldern von comapnies zu tun zu haben.

Reply:
@dennis-dfine

Reply:
I think the change from fiscal year end to level of assurance was because this test depended on fiscal year end already existing, which is not always the case in the fake fuxtures. I think @dennis-dfine tried to make that consistent via the fake fixture generator but couldn't?

The changes below it are simply formatting